README.md

```diff
@@ -1,6 +1,29 @@
-# Eclipse GLSP Examples [![build-status](https://img.shields.io/jenkins/build?jobUrl=https%3A%2F%2Fci.eclipse.org%2Fglsp%2Fjob%2Feclipse-glsp%2Fjob%2Fglsp-examples%2Fjob%2Fmaster%2F)](https://ci.eclipse.org/glsp/job/eclipse-glsp/job/glsp-examples/job/master) [![build-status-server](https://img.shields.io/jenkins/build?jobUrl=https://ci.eclipse.org/glsp/job/deploy-npm-glsp-examples/&label=npm)](https://ci.eclipse.org/glsp/job/deploy-npm-glsp-examples) [![build-status-server](https://img.shields.io/jenkins/build?jobUrl=https://ci.eclipse.org/glsp/job/deploy-m2-glsp-examples/&label=maven)](https://ci.eclipse.org/glsp/job/deploy-m2-glsp-examples)
+# Eclipse GLSP Examples ![build-status](https://img.shields.io/jenkins/build?jobUrl=https%3A%2F%2Fci.eclipse.org%2Fglsp%2Fjob%2Feclipse-glsp%2Fjob%2Fglsp-examples%2Fjob%2Fmaster%2F)
+
+This repository contains code examples that demonstrates how build diagram editors with the [Graphical Language Server Platform (GLSP)](https://github.com/eclipse-glsp/glsp). The examples are focused on the integration of GLSP editors with the cloud-based [Eclipse Theia IDE](https://github.com/theia-ide/theia) using the [GLSP theia integration](https://github.com/eclipse-glsp/glsp-theia-integration).
+
+Each examples is self-contained and provides both, a example diagram client and its corresponding GLSP server. 
+
+## Prerequisites
+The following libraries/frameworks need to be installed on your system:
+- [Node.js](https://nodejs.org/en/) `>= 12.14.1 AND < 13`
+- [Yarn](https://classic.yarnpkg.com/en/docs/install#debian-stable) `>=1.7.0`
+- [Java](https://www.oracle.com/java/technologies/javase-jdk11-downloads.html) `>=11`
+- [Maven](https://maven.apache.org/) `>=3.6.0`
+
+ The examples are heavily interweaved with Eclipse Theia, so please also check the [prerequisites of Theia](https://github.com/eclipse-theia/theia/blob/master/doc/Developing.md#prerequisites).
+
+The web-based/client part of the examples has been developed using [Visual Studio Code](https://code.visualstudio.com/) and the server/java part has been developed with the [Eclipse IDE](https://www.eclipse.org/ide/). However, it's of course also possible to use any other IDE or text editor.
+
+## Usage
+- `git clone https://github.com/eclipse-glsp/glsp-examples.git`
+- `cd <any-example-folder>`
+- `cd client && yarn && cd ..` (build client)
+- `cd server && mvn clean install && cd ..` (build server)
```

Comment:
This step is not correct. When I perform it I get
```
[ERROR] The goal you specified requires a project to execute but there is no POM in this directory (/home/martin/Temp/GLSP/glsp-examples/minimal/server). Please verify you invoked Maven from the correct directory. -> [Help 1]
```
and indeed there is no pom.xml in the server directory so I probably need to go into `org.eclipse.glsp.example.minimal` inside the server directory. Build there provides me (among some other things the following files:
- org.eclipse.glsp.example.minimal-0.0.1-SNAPSHOT-glsp.jar
- org.eclipse.glsp.example.minimal-0.0.1-SNAPSHOT-sources.jar
- org.eclipse.glsp.example.minimal-0.0.1-SNAPSHOT.jar

While it is obvious to me what the SNAPSHOT and SNAPSHOT-sources is, what is the SNAPSHOT-glsp.jar?

Maybe, if you want to keep the generic 'Usage' section as is, the additional information could go into the "Minimal Example" section below?

Reply:
Snapshot-glsp is the exectuable shaded maven bundle (fat-jar) that contains all dependencies and can be used to start the server via commandline.

I'm planning on providing additional details for running/building the examples soon. However, I'm planning on simplifying the start/build process  (similar to how it was done in the coffee editor) and once this is done I will update the README accordingly. For now I just wanted to make sure that the main README is (at least sort-of) up-to-date.